js/src/personal/riyuan/components/navigation/Navigation.module.css

```diff
@@ -40,6 +40,12 @@
   width: 88px;
 }
 
+.header{
+  padding-top: 5px;
+  display: flex;
+}
+
+
 /************************************   */
 /* item list */
 /************************************   */
```

Comment:
nit: feel free to do late, but once these sections or this file gets large, you should split it into even smaller css modules